packages/gestalt/src/Borders.css

```diff
@@ -48,8 +43,40 @@ html[dir="rtl"] .borderLeft {
   border-radius: 999px;
 }
 
-.rounded {
-  border-radius: var(--border-radius);
+.rounding0 {
+  border-radius: 0;
```

Comment:
@jennyscript instead, could we set these to:

```css
border-radius: calc(var(--bt) * 0);
border-radius: calc(var(--bt) * 1);
```
It makes it more clear that we're using boints for these increases.

Also see `packages/gestalt/src/boxWhitespace.css`